src/tools/perf/lib/libperf_int.h

```diff
@@ -30,6 +30,7 @@ BEGIN_C_DECLS
 #define UCT_PERF_TEST_AM_ID  5
 #define ADDR_BUF_SIZE        4096
 #define EXTRA_INFO_SIZE      256
+#define UCX_PERF_DEV_DEFAULT -1
```

Comment:
1. UCX_PERF_MEM_DEV_DEFAULT
2. IMO it should be in libperf.h because it is passed in ucx_perf_params